### PR TITLE
feat: add DNAME and fix healthcheck support

### PR DESCRIPTION
**Pull request by jerjako**

### CODE REVIEW

File: octodns_scaleway/__init__.py [hunk 1]
```diff
@@ -206,6 +207,37 @@ def _data_dynamic_weight(self, weighted_config):
...
+        pools = {
+            'pool-0': {
+                'values': []
+            }
+        }
+        rules = [{
+            'pool': 'pool-0'
+        }]
+
+        values = []
```
Comment (remyleone) [inline octodns_scaleway/__init__.py:220]: Could list comprehension work here?

Comment (jerjako) [reply octodns_scaleway/__init__.py]: yep it's better, good catch

File: octodns_scaleway/__init__.py [hunk 2]
```diff
@@ -206,6 +207,33 @@ def _data_dynamic_weight(self, weighted_config):
             'rules': rules
         }
 
+    def _data_dynamic_healthcheck(self, http_service_config):
```
Comment (remyleone) [inline octodns_scaleway/__init__.py:210]:
```
    def _data_dynamic_healthcheck(self, http_service_config):
        url = urlparse(http_service_config['url'])
        return {
            'pools': {
                'pool-0': {
                    'values': [{'value': ip} for ip in http_service_config['ips']]
                }
            },
            'rules': [{
                'pool': 'pool-0'
            }],
            'octodns': {
                'healthcheck': {
                    'host': url.hostname,
                    'path': url.path,
                    'port': url.port,
                    'protocol': url.scheme.upper()
                }
            }
        }

```

Comment (jerjako) [reply octodns_scaleway/__init__.py:210]: :pray:

**Review by remyleone — APPROVED** [review]

**Pull request merged**